src/test/java/mate/academy/springboot/datajpa/DataJpaApplicationTests.java

```diff
@@ -1,13 +0,0 @@
-package mate.academy.springboot.datajpa;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class DataJpaApplicationTests {
-
-    @Test
-    void contextLoads() {
-    }
-
-}
```

Comment:
Would be nice to let it stay. A basic test of loading context is always nice to have